### PR TITLE
Manual attempt of template sync for 3.0.2

### PR DESCRIPTION
**Pull request by jfy133**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/mag/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/mag _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -915,49 +853,49 @@
...
-            "$ref": "#/definitions/reference_genome_options"
+            "$ref": "#/$defs/reference_genome_options"
         },
         {
-            "$ref": "#/definitions/institutional_config_options"
+            "$ref": "#/$defs/institutional_config_options"
         },
         {
-            "$ref": "#/definitions/max_job_request_options"
+            "$ref": "#/$defs/max_job_request_options"
```
Comment (nvnieuwk) [inline nextflow_schema.json:865]: This should be removed from the schema

**Comment by jfy133** [comment]
TODO: 

- check input validation still working against broken sheets: [borked.zip](https://github.com/user-attachments/files/17413709/borked.zip)

File: CITATIONS.md
Comment (mashehu) [inline CITATIONS.md:1]: the original indentation was correct

File: nextflow.config [hunk 2]
```diff
@@ -150,7 +150,7 @@ params {
     save_mmseqs_db                       = false
 
     // References
-    genome                     = null
+    //genome                     = null // we use --host_genome instead
```
Comment (mashehu) [inline nextflow.config:153]:
```suggestion
```
or do you want to explicitly document it?

File: nextflow_schema.json [hunk 3]
```diff
@@ -586,7 +523,7 @@
                 },
                 "gtdbtk_pplacer_cpus": {
                     "type": "number",
```
Comment (mashehu) [inline nextflow_schema.json:525]: why not use integers to count CPUs?

**Review by mashehu — APPROVED** [review]

File: nextflow.config (hunk 2 above)
Comment (jfy133) [reply nextflow.config:153]: Easier for template merges to remember to 'deactivate it'

File: nextflow_schema.json (hunk 3 above)
Comment (jfy133) [reply nextflow_schema.json]: Fixed! I dunno why it was a number... might be very old left over

File: nextflow.config (hunk 2 above)
Comment (mashehu) [reply nextflow.config:153]: not anymore for future template updates thanks to the `skip_features: [igenomes]`

**Pull request merged**